.circleci/config.yml

```diff
@@ -416,36 +358,34 @@ jobs:
             pipenv run pytest --cov=nucypher --cov-report xml:reports/coverage.xml $(circleci tests glob "tests/network/**/test_*.py" | circleci tests split --split-by=timings)
       - capture_test_results
 
-  keystore:
+  character:
     <<: *python_36_base
-    parallelism: 2
+    parallelism: 4
     steps:
       - prepare_environment
       - run:
-          name: Keystore Tests
+          name: Character Tests
           command: |
-            pipenv run pytest --cov=nucypher --cov-report xml:reports/coverage.xml $(circleci tests glob "tests/keystore/**/test_*.py" | circleci tests split --split-by=timings)
+            pipenv run pytest --cov=nucypher --cov-report xml:reports/coverage.xml $(circleci tests glob "tests/characters/**/test_*.py" "tests/learning/**/test_*.py" | circleci tests split --split-by=timings)
       - capture_test_results
 
-  character:
+  cli:
     <<: *python_36_base
-    parallelism: 4
+    parallelism: 6
     steps:
       - prepare_environment
       - run:
-          name: Character Tests
+          name: Nucypher CLI Tests
           command: |
-            pipenv run pytest --cov=nucypher --cov-report xml:reports/coverage.xml $(circleci tests glob "tests/characters/**/test_*.py" | circleci tests split --split-by=timings)
+            pipenv run pytest --cov=nucypher --cov-report xml:reports/coverage.xml $(circleci tests glob "tests/cli/**/test_*.py" | circleci tests split --split-by=timings)
       - capture_test_results
 
-  learning:
+  tests_ok:
```

Comment:
Also interesting - Why the extra step?  Is the green / red success not enough indication, or is this a matter of making the "fan-in" logic more maintainable?

Reply:
Yeah, it's purely cosmetic, to collect all test jobs in a single entry point for subsequent jobs.